src/__test__/components/data-exploration/differential-expression-tool/DiffExprCompute.test.jsx

```diff
@@ -334,4 +334,66 @@ describe('DiffExprCompute', () => {
       expect(screen.getByText(/Compute/i).closest('button')).not.toBeDisabled();
     });
   });
+
+  it('Unselect deleted option', async () => {
```

Comment:
Does this test make sense now that we have removed the code handling deletion on the react component?

Reply:
No, the test still stand, because it's testing for the expected outcome, not the functionality.

Reply:
So it is testing the redux action?

Reply:
Yes